code/modules/vehicles/vehicle_actions.dm

```diff
@@ -233,6 +233,6 @@
 			V.pass_flags &= ~PASSTABLE
 		if(locate(/obj/structure/table) in V.loc.contents)
 			V.grinding = TRUE
-			V.icon_state = "[V.board_icon]-grind"
+			V.icon_state = "[initial(V.icon_state)]-grind"
```

Comment:
Not gonna consider this one a merge blocker, but if you have the time I'd love to see all the Vs and Ls replaced as above.

Reply:
might as well while I'm here